src/GifPickerReact.tsx

```diff
@@ -27,7 +28,7 @@ export interface GifPickerReactProps {
 
 function GifPickerReact(props: GifPickerReactProps): JSX.Element {
 	const settings = useSettings(props);
-	const pickerContext = usePickerContext();
+	const pickerContext = usePickerContext(props.initialSearchTerm);
```

Comment:
I would pass this property through `useSettings` and set default there

Reply:
I considered that, but we need to call the usePickerContext inside the <SettingsContext>. is that a problem for you? I can change that.

Reply:
Why not something like that? `settings` is just an object
```ts
const settings = useSettings(props);
const pickerContext = usePickerContext(settings.initialSearchTerm);
```

Reply:
Done